packages/formation-react/src/components/ErrorableSelect/ErrorableSelect.jsx

```diff
@@ -82,7 +82,7 @@ class ErrorableSelect extends React.Component {
         </label>
         {errorSpan}
         <select
-          className={this.props.selectClass || this.props.additionalClass}
```

Comment:
Well that's annoying. I'd prefer `selectClass` to `additionalClass` myself, but I understand not changing the usage in the wild right now.

Reply:
There are similar inconsistencies like this scattered throughout. It isn't a high priority, but if one day we find ourselves with not much to do (:laughing: ) we could go through and make prop names more consistent.